src/ui.py

```diff
@@ -161,6 +161,10 @@ def load_values_from_dict(window, values, ignorelist=None):
                 widget.set_active(bool(value))
             elif isinstance(widget, Adw.ExpanderRow):
                 widget.set_enable_expansion(bool(value))
+            elif isinstance(widget, Gtk.Scale):
```

Comment:
Generally, a whole bunch of "if isintance" checks are not really a great practice, but in this case I think it makes sense. And again, a big improvement over settings all possible properties by hand, this really simplifies things a lot.